src/TestFramework/TestFramework/Assertions/Assert.AreEqual.cs

```diff
@@ -51,7 +51,7 @@ public static void AreEqual<T>(T? expected, T? actual)
     /// </param>
     /// <param name="comparer">
     /// The <see cref="System.Collections.Generic.IEqualityComparer{T}"/> implementation to use when comparing keys,
-    /// or null to use the default <see cref="System.Collections.Generic.EqualityComparer{T}"/>.
+    /// or null to use the default <see cref="System.Collections.Generic.EqualityComparer{T}"/>./>.
```

Comment:
This looks like a XmlDoc syntax error. I'm wondering why this doesn't raise a build warning on the CI build.